shared-libs/cht-datasource/test/index.spec.ts

```diff
@@ -124,6 +125,24 @@ describe('CHT Script API - getDatasource', () => {
         expect(personGet.calledOnceWithExactly(qualifier)).to.be.true;
         expect(byUuid.calledOnceWithExactly(qualifier.uuid)).to.be.true;
       });
+
+      it('getPage', async () => {
+        const expectedPeople: Index.Nullable<Doc>[] = [];
```

Comment:
Minor, the type here can be `Person.v1.Person[]` instead of `Index.Nullable<Doc>[]`.